Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.7-alpine
 LABEL maintainer="Jacob Tomlinson <jacob@tom.linson.uk>"
 
-RUN apk add --no-cache gcc musl-dev
+RUN apk add --no-cache gcc musl-dev alpine-sdk
```

Comment:
Do we need this now that we aren't adding the dependency?

Reply:
Actually we don't, i thought this was a dependency that we added with the gcc, my bad! 
I'll update the docker file 😄

Reply:
Thinking about it we may need it for the tests. Let's see how the CI goes.